src/main/scala/org/clulab/reach/ReachCLI.scala

```diff
@@ -55,6 +57,27 @@ class ReachCLI (
 
   def doAssembly (mns: Seq[Mention]): Assembler = Assembler(mns)
 
+  def deserializeDoc(file: File) = {
+    val br = new BufferedReader(new InputStreamReader(new FileInputStream(file)))
+    val entryStr = new StringBuilder()
+
+    var line = br.readLine().trim
+    while (line != "#FS#") {
+      entryStr ++= line
+      line = br.readLine().trim
+    }
+
+    val tokens = entryStr.toString.split('\t')
+    val entry = FriesEntry.mkFriesEntry(tokens.head, tokens.last)
```

Comment:
When is mkFriesEntry used vs. fromString?

Reply:
`fromString` will be removed